RELEASE-NOTES.md

```diff
@@ -1,5 +1,74 @@
+- Improve IDE support for `PrefectObjectRegistry.register_instances` decorated classes — https://github.com/PrefectHQ/prefect/pull/11617
+- Make the UI accessible via reverse proxy and add a `--no-install` flag to `prefect dev build-ui` — https://github.com/PrefectHQ/prefect/pull/11489
+- Improve UI build during `prefect server start` - https://github.com/PrefectHQ/prefect/pull/11493
+- Improve error message in `.deploy` — https://github.com/PrefectHQ/prefect/pull/11615
+
+### Fixes
+- Use default values (if any) when no run input is provided upon `resume` — https://github.com/PrefectHQ/prefect/pull/11598
+- Prevent deployments with `RRule` schedules containing `COUNT` — https://github.com/PrefectHQ/prefect/pull/11600
+- Fix flows with class-based type hints based on `from __future__ import annotations` — https://github.com/PrefectHQ/prefect/pull/11578 & https://github.com/PrefectHQ/prefect/pull/11616
+- Raise `StepExecutionError` on non-zero `run_shell_script` return code during `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/11604
+
+### Experimental
+- Enable flow runs to receive typed input from external sources — https://github.com/PrefectHQ/prefect/pull/11573
+
+### Documentation
+- Fix non-rendering link in Docker guide — https://github.com/PrefectHQ/prefect/pull/11574
+- Update deployment and flow concept docs — https://github.com/PrefectHQ/prefect/pull/11576
+- Add examples for custom triggers to automations docs — https://github.com/PrefectHQ/prefect/pull/11589
+- Add send/receive documentation to `run_input` module docstring — https://github.com/PrefectHQ/prefect/pull/11591
+- Add automations guide — https://github.com/PrefectHQ/prefect/pull/10559
+- Fix storage guide links and reference — https://github.com/PrefectHQ/prefect/pull/11602
+- Fix typo in `prefect deploy` guide — https://github.com/PrefectHQ/prefect/pull/11606
+- Fix imports in human-in-the-loop workflows guide example — https://github.com/PrefectHQ/prefect/pull/11612
+- Add missing imports to human-in-the-loop workflows example — https://github.com/PrefectHQ/prefect/pull/11614
+- Fix formatting in `prefect-deploy` guide — https://github.com/PrefectHQ/prefect/pull/11562
+- Remove "Notification blocks must be pre-configured" warning from automations docs — https://github.com/PrefectHQ/prefect/pull/11569
+- Update work pools concept docs example to use correct entrypoint — https://github.com/PrefectHQ/prefect/pull/11584
+
+### New Contributors
+- @brett-koonce made their first contribution in https://github.com/PrefectHQ/prefect/pull/11562
+- @jitvimol made their first contribution in https://github.com/PrefectHQ/prefect/pull/11584
+- @oz-elhassid made their first contribution in https://github.com/PrefectHQ/prefect/pull/11353
+- @Zyntogz made their first contribution in https://github.com/PrefectHQ/prefect/pull/11415
+- @Andrew-S-Rosen made their first contribution in https://github.com/PrefectHQ/prefect/pull/11578
+
```

Comment:
I think we're missing the diff that usually sits here

Reply:
https://github.com/PrefectHQ/prefect/pull/11622/files#r1449441310 will resolve it